.travis.yml

```diff
@@ -42,6 +42,7 @@ script:
  # - bazel clean && bazel build //...
  - bazel test reconciletags:reconciletags_test
  - bazel run //ftl:node_builder_test
```

Comment:
I don't think this one needs a "bazel run", could you combine these two to "bazel test //ftl:..."